src/com/rcv/TieBreak.java

```diff
@@ -128,14 +129,15 @@ private String breakTie() {
   // return: candidateID of the selected loser
   private String doInteractive() {
     System.out.println(
-      "Tie in round " + round + " for the following candidateIDs each of whom has " + numVotes + " votes:"
+        "Tie in round " + round + " for the following candidateIDs each of whom has " + numVotes
+            + " votes:"
     );
     // i: index over tied candidates
     for (int i = 0; i < tiedCandidates.size(); i++) {
-      System.out.println((i+1) + ". " + tiedCandidates.get(i));
+      System.out.println((i + 1) + ". " + tiedCandidates.get(i));
     }
     System.out.println(
-      "Enter the number corresponding to the candidate who should lose this tiebreaker."
+        "Enter the number corresponding to the candidate who should lose this tiebreaker."
```

Comment:
Not for this PR, but we need to add support for interactive tiebreaking in the GUI.

Reply:
Added issue #49.